test-apps/remix-vite-embedded/README.md

```diff
@@ -0,0 +1,28 @@
+# templates/unstable-vite
+
+⚠️ Remix support for Vite is unstable and not recommended for production.
```

Comment:
how old of a template did you use here? 🤣

Reply:
It's just a copy of `test-apps/remix-vite` 😅

Reply:
Oh boy thats embarrassing 😳